### PR TITLE
Update event-013.md

### PR DESCRIPTION
**Pull request by catsmith**
adding times

It looks like I have to do a separate pull request for each event. If there is another way to do this please let me know.

### CODE REVIEW

File: _events/discussions/event-013.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ affiliations:
       index: 2
     - name: Stanford University, US
       index: 3
+times:
+    - start: 2020-09-16T18:00:00Z
+    - end: 2020-09-16T19:30:00Z
```
Comment (Chilipp) [inline _events/discussions/event-013.md:26]:
```suggestion
time:
    - start: 2020-09-16T18:00:00Z
      end: 2020-09-16T19:30:00Z
```

the correct syntax is 

```yaml
time:
    - start: isoformatted-time
      end: isoformatted-time
```

**Review by Chilipp — CHANGES_REQUESTED** [review]

**Comment by catsmith** [comment]
Ah okay. I will do that in future. These was a disparity between the video and the example you sent as a guide and I went with the wrong one!

**Review by Chilipp — APPROVED** [review]

**Comment by Chilipp** [comment]
alright, no worries @catsmith. Thanks a lot for your great job here :smiley: 

From my perspective, this is ready to be merged. Are you okay with that?

PS: Here are some previews:
- https://980-267395254-gh.circle-artifacts.com/0/SORSE/programme/index.html
- https://980-267395254-gh.circle-artifacts.com/0/SORSE/programme/calendar/index.html
- https://980-267395254-gh.circle-artifacts.com/0/SORSE/programme/discussions/event-013/index.html

**Comment by catsmith** [comment]
Looks great! I think everything apart from the one in draft can be live
now.

This format of adding (now I know how to do it correctly) is fine for me
for future scheduling too.

**Comment by Chilipp** [comment]
great, thanks :smiley:

**Pull request merged**